### PR TITLE
😎 Added my card

### PR DESCRIPTION
**Pull request by raj-pulapakura**
This is my first contribution to any GitHub repo. Thanks for the opportunity!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -107,6 +107,33 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+                <!-- ________ Raj-Pulapakura card START ________  -->
+                <div class="card">
+                  <p class="name">Raj Pulapakura</p>
+                  <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="https://www.twitter.com/" target="_blank">I don't use Twitter lol</a>
```
Comment (TenTraicion) [inline index.html:115]:
```suggestion
```

Remove these 2 line if you don't use twitter.

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@raj-pulapakura, kindly make the required change to merge it successfully. Thanks!

File: index.html (hunk 1 above)
Comment (raj-pulapakura) [reply index.html:115]: Thanks for the tip.

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@raj-pulapakura, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**